src/cls/rgw/cls_rgw_client.cc

```diff
@@ -24,8 +24,8 @@ class ClsBucketIndexOpCtx : public ObjectOperationCompletion {
   int *ret_code;
 public:
   ClsBucketIndexOpCtx(T* _data, int *_ret_code) : data(_data), ret_code(_ret_code) { assert(data); }
-  ~ClsBucketIndexOpCtx() {}
-  void handle_completion(int r, bufferlist& outbl) {
+  ~ClsBucketIndexOpCtx() override {}
```

Comment:
It's inserted automatically. I think it's useful because of that its parent class decl a virtual deconstructor.